articles/best-javascript-animation-libraries/index.md

```diff
@@ -0,0 +1,451 @@
+      }
+
+      .circle {
+        background: #78db74;
+        width: 50px;
+        height: 50px;
+        border-radius: 100%;
+      }
+    </style>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/gsap/3.5.1/gsap.min.js"></script>
+    <title>GreenSockJs Animation</title>
+  </head>
+  <body>
+    <div class="circle"></div>
+    <script>
+      gsap.to(".circle", {
+        duration: 3,
+        x: 1000,
+        scale: 4,
+      });
+    </script>
+  </body>
+</html>
+```
+### Popmotion.js
+
+[Popmotion.js](https://popmotion.io/) is a lightweight and robust JavaScript animation library. It is powerful because it supports animations like Keyframes, spring, decay, and complex animations.
+Popmotion.js makes it easy for front-end developers and web designers to animate 3D objects, React components, SVG, CSS, and DOM elements.
+
+### Typed.js
+
+[Typed.js](https://mattboldt.com/demos/typed-js/) is a JavaScript animation library that provides the experience of creating a typing animation on your webpage. It has various options to modify how the animations function.
+Typed.js options include; strings, type speed, start delay, loop, back speed, and back delay.
+
+To get started with type.js, download the latest version of the animation library from GitHub. Then include the script to your page using a script tag.
+```html
+ <script src="https://cdn.jsdelivr.net/npm/typed.js@2.0.11"></script>
+  <script src="typed.js"></script>
+  ```
+  Type.js example
+
+```html
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Typed.js Animation</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <div class="center">
+      <p>This is <span class="animate"></span></p>
+    </div>
+  </body>
+  <script src="https://cdn.jsdelivr.net/npm/typed.js@2.0.11"></script>
+  <script src="typed.js"></script>
+  <script>
+    var typed = new Typed(".animate", {
+      strings: [
+        "Section Community",
+        "typed.js Animation, It creates typing animations on a web page",
+      ],
+      typeSpeed: 40,
+      backDelay: 50,
+      backSpeed: 30,
+      loop: true,
+    });
+  </script>
+</html>
+```
+
+Here is type.js style.css file
+
+```css
+body {
+  margin: 0;
+  padding: 0;
+  background: #8b008b;
+}
+.center {
+  top: 55%;
+  text-align: center;
+  font-family: Arial, Helvetica, sans-serif;
+  font-weight: 200;
+  font-size: large;
+  width: 100%;
+}
+```
+### Conclusion
+
+There are very many JavaScript animation libraries that can be implemented on your projects. Those listed above are a few with the best combination of complexity, ease, and stability. Each animation library is different from another and suites many different situations.
+Suppose you are looking for a powerful animation. The best JavaScript library option to go for is Anime.js, Velocity.js, GreenSock.js, and Popmotion.js.
```

Comment:
"The best JavaScript library options to go for" instead of "The best JavaScript library option to go for"